packages/client/mutations/EndRetrospectiveMutation.ts

```diff
@@ -28,6 +28,13 @@ graphql`
       reflectionCount
       taskCount
       topicCount
+      facilitator {
```

Comment:
Checking the facilitator feature flag instead of the `summary` field in the retro meeting because it may not have been generated by the time the meeting ends